app/javascript/controllers/style_labels_controller.js

```diff
@@ -0,0 +1,19 @@
+import { Controller } from "@hotwired/stimulus"
+
+// Connects to data-controller="style-labels"
+export default class extends Controller {
+  static targets = [ "checkBox", "label" ]
+  static classes = [ "background", 'text' ]
```

Comment:
Use single qoutes for strings: https://github.com/airbnb/javascript?tab=readme-ov-file#strings--quotes